load-test/src/test/scala/com/mainflux/loadtest/PublishHttpMessages.scala

```diff
@@ -1,7 +1,5 @@
 /*
- * Copyright (c) 2018
- * Mainflux
- *
+ * Copyright (c) 2018 Mainflux *
```

Comment:
Please remove the date from this comment.

Reply:
Shoot I missed one.  I had to double check them all.  Looks like this was the only one that slipped by.